### ISSUE TITLE
[Feature request] Reserve copies of files

### ISSUE THREAD

**Issue by thima2017** [NatronGitHub/Natron#562]
### Problem

My 2 hours work is lost because I have a bad computer or something (it just turned off). My Natron file I was working on is 0 kb now.

**Expected behavior:** 

I expected to find any of a previous versions in a cache or somewhere.

**Actual behavior:** 

I don't find any way to recover any part of my work.
I guess Natron should have something like this (screenshot from Blender Preferences > File & Load. It makes Blender to save a copies of the current file in the same directory with the original file):
![Untitled](https://user-images.githubusercontent.com/32433505/106349635-88aa7180-62e0-11eb-9ed6-7e6c5f89daff.png)

### Versions

* **Natron version/commit** (they can be retrieved from the About Window): Natron version 2.3.15 Release.
* **OS version:** Ubuntu 20.04

**Comment by rodlie** [comment]
how is this suppose to work? Make duplicates every time you hit save?

I always saves increments of what I do, when I hit a "milestone" in my project I save to a new file (x02.ntp, x03.ntp, x04.ntp etc).

**Comment by thima2017** [comment]
Yes. This is how it works in Blender.
Currently I use this bash script:
```
while true
do
sleep 5m
cp -f ./baks/render5.ntp2 ./baks/render5.ntp3
cp -f ./baks/render5.ntp1 ./baks/render5.ntp2
cp -f ./render5.ntp ./baks/render5.ntp1
date
done
```

**Comment by devernay** [comment]
implemented in #740

**Comment by devernay** [comment]
will be in Natron 2.4.3/2.5.0
